### ISSUE TITLE
Create Event - Aria-label warning

### ISSUE THREAD

**Issue by EricSSartorius** [EricSSartorius/Judging-System#15]
Aria-Label warning shown in console on Create Event page after adding Angular-Material to form.

`ARIA: Attribute " aria-label ", required for accessibility, is missing on node: input.ng-pristine.ng-untouched.ng-valid   angular_angular.js:12485 ARIA: Attribute "`

**Comment by EricSSartorius** [comment]
Material Design input tags didn't have aria-label automatically built in so I just had to add it to each input tag using MD.
